index.html

```diff
@@ -2457,23 +2393,26 @@ <h3><dfn>Writing or pushing content</dfn></h3>
                 </dl>
               </li>
               <li>
-                Let |transformedPromise| be the result of
-                <a data-cite="promises-guide">transforming</a> |p| with a fulfillment
-                and rejection handler that:
+                [=promise/React=] to |p|:
                 <ol>
                   <li>
-                    Clears the <a>pending push tuple</a>.
-                  </li>
-                  <li>
-                    Removes <a>async write handlers</a>.
+                    If |p| was settled (fulfilled or rejected), then:
+                    <ol>
+                      <li>
+                        Clears the <a>pending push tuple</a>.
```

Comment:
I believe you want "Clear" and "Remove".

Reply:
@zolkis Does this suggestion make sense to you?

Reply:
I think that was the original intent of the text (assuming removal, too). For safety, will let @kenchris clarify.

Reply:
I guess the clarification is warranted, since we don't have any explicit steps for "clear the pensing push tuple".